tests/__init__.py

```diff
@@ -27,8 +27,13 @@ def __init__(self, path, config_data=None, *args, **kwargs):
         self.path = os.path.join(os.path.abspath(path), 'project')
         self.config_file = os.path.join(self.path, '.therapist.yml')
 
-        sample_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'sample_project')
-        shutil.copytree(sample_path, self.path)
+        blank = kwargs.get('blank', False)
```

Comment:
Why not just put blank in the parameter list like `def __init__(self, path, config_data=None, *args, blank=false, **kwargs):`?